src/torch/memory.c

```
@@ -0,0 +1,144 @@
+#define TORCH_MEMORY_ALIGN 64
+
+static size_t align_up(size_t n, size_t align) {
+    return (n + align - 1) & ~(align - 1);
+}
+
+TorchMemoryOptions torch_memory_default_options(void) {
+    TorchMemoryOptions opts = {.alignment = TORCH_MEMORY_ALIGN, .track_peak = true};
+    return opts;
+}
+
+TorchMemoryManager* torch_memory_create(size_t size) {
```

Cppcheck (2.21.0)

[style] The function 'torch_memory_create' is never used.

(unusedFunction)